tests/network/vmi_istio.go

```diff
@@ -67,6 +67,8 @@ const (
 	// Istio uses certain ports for it's own purposes, this port server to verify that traffic is not routed
```

Comment:
commit message nit (only if you need to rebase for something else)

` tests, network: Move an instio const closer to its usage `

instio->istio

Reply:
Thanks for noticing, done.